#goal
float32[] pose_b
---
#result
float32[] obstacles
---
#feedback
string status
